rust/connlib/tunnel/src/dns.rs

```diff
@@ -674,6 +694,52 @@ mod tests {
         assert_eq!(response.response_code(), ResponseCode::NXDOMAIN);
         assert_eq!(response.records().count(), 0);
     }
+
```

Comment:
I seem to remember we did something with HTTP(s) queries too, do we need to account for that here? IIRC it was to block them so that we can intercept the A or AAAA equivalents sent by the browsers as fallback or something.

Reply:
Yeah that is handled separately because it is a different query type.